src/components/input/input.component.tsx

```diff
@@ -134,34 +140,44 @@ const Input = React.forwardRef<RNTextInput, InputProps>(
       }
     };
 
+    const onPressComponent = () => {
+      if (innerRef?.current) {
+        innerRef?.current?.focus();
+      }
+    };
+
     return (
-      <RNView style={computedStyle.container}>
-        {prefix && <RNView style={computedStyle.prefix}>{prefix}</RNView>}
-        <RNTextInput
-          ref={ref}
-          onFocus={onFocusInput}
-          onBlur={onBlurInput}
-          selectionColor={
-            typeof props.selectionColor === 'string'
-              ? getThemeColor(theme.colors, props.selectionColor)
-              : props.selectionColor
-          }
-          {...rest}
-          style={computedStyle.input}
-          placeholderTextColor={placeholderColor}
-        />
-        {!isLoading && suffix && (
-          <RNView style={computedStyle.suffix}>{suffix}</RNView>
-        )}
-        {isLoading && (
-          <RNView style={computedStyle.suffix}>
-            <RNActivityIndicator
-              size={getThemeProperty(theme.fontSize, loaderSize)}
-              color={getThemeColor(theme.colors, loaderColor)}
+      <TouchableWithoutFeedback onPress={onPressComponent}>
+        <RNView style={computedStyle.marginContainer}>
+          <RNView style={computedStyle.container}>
+            {prefix && <RNView style={computedStyle.prefix}>{prefix}</RNView>}
+            <RNTextInput
+              ref={innerRef}
+              onFocus={onFocusInput}
+              onBlur={onBlurInput}
+              selectionColor={
+                typeof props.selectionColor === 'string'
+                  ? getThemeColor(theme.colors, props.selectionColor)
+                  : props.selectionColor
+              }
+              {...rest}
+              style={computedStyle.input}
+              placeholderTextColor={placeholderColor}
```

Comment:
On veut vraiment ça ?  
Est-ce qu'il faut pas plutôt faire quelque chose comme ça ?  
```suggestion
              placeholderTextColor={placeholderColor}
              {...rest}
              style={{ ...computedStyle.input, ...rest.style }}
```

Reply:
Yes change made on the lines, but we can keep style as it is, because `rest.style` is already computed into `computedStyle.container`